src/main/resources/META-INF/configurations/org.jahia.modules.richtext-ckeditor5.cfg

```diff
@@ -0,0 +1,6 @@
+enabledByDefault=true
```

Comment:
We probably want to add `# default configuration` at the top here to not override user changes on every deploy